tools/index.d.ts

```diff
@@ -0,0 +1,168 @@
+import { RawSourceMap } from 'source-map';
```

Comment:
There is no `index.js` file in `tools`; this file should be named `tools/node.d.ts`.